student.rb

```diff
@@ -0,0 +1,8 @@
+class Student
+  attr_accessor :name, :classroom
```

Comment:
[OPTIONANL BUT RECOMMANDED] Please make a relationship between Student and Person for now you did relate your Student to Person so kindly fix this issue :slightly_smiling_face: